firefox-ios/firefox-ios-tests/Tests/XCUITests/SyncFAUITests.swift

```diff
@@ -73,7 +73,7 @@ class SyncUITests: BaseTestCase {
         // Enter only email, wrong and correct and tap sign in
         userState.fxaUsername = "foo1bar2baz3@gmail.com"
         navigator.performAction(Action.FxATypeEmail)
-        navigator.performAction(Action.FxATapOnSignInButton)
+        navigator.performAction(Action.FxATapOnContinueButton)
```

Comment:
@dragosb01 What do you think?

Reply:
It works for me. @isabelrios this PR can be merged as it is.